analyses/fusion-frequencies/README.md

```diff
@@ -16,26 +16,35 @@ Each gene invloved in the fusion is annotated by a Gene_Position, for example:
 
 #### Frequency annotation
 
-Each `cancer_group` and `cohort`(s) combination is considered a `cancer_group_cohort`. `cancer_group_cohort` with `n_samples` >= 5, compute `Frequency_in_overall_dataset`, `Frequency_in_primary_tumors`, and `Frequency_in_relapse_tumors` as following:
+For each `cancer_group` and `cohort`(s) combination is considered a `cancer_group_cohort`. `cancer_group_cohort` with `n_samples` >= 5,  `Frequency_in_overall_dataset`, `Frequency_in_primary_tumors`, and `Frequency_in_relapse_tumors` were computed. 
+In this module, we generated two fusion frequency tables - one `putative-oncogene-fusion-freq.jsonl.gz` calculate the frequency of a specific fusion (with both partners) in primary tumors, relapse tumors and overall dataset. 
```

Comment:
```suggestion
In this module, we generated two fusion frequency tables - one `putative-oncogene-fusion-freq.jsonl.gz` calculate the frequency of a specific fusion in primary tumors, relapse tumors, and overall dataset. 
```

just updating this since one breakpoint or both can be intergenic (and a big fan of the oxford comma 😆 )